src/common/dpi_aware.cpp

```diff
@@ -14,6 +14,18 @@ HRESULT DPIAware::GetScreenDPIForWindow(HWND hwnd, UINT &dpi_x, UINT &dpi_y) {
   }
 }
 
+HRESULT DPIAware::GetScreenDPIForPoint(POINT p, UINT& dpi_x, UINT& dpi_y) {
+	auto monitor_handle = MonitorFromPoint(p, MONITOR_DEFAULTTOPRIMARY);
```

Comment:
Please replace tabs with spaces.
In some files we use 2 spaces for indentation, in other we use 4. It's unfortunate that we didn't adopt a common indentation for the entire project, but at least let's try to preserve each file indentation.
Thank you.

Reply:
Thanks for pointing that out! I try to stay consistent within the file. Sorry for overseeing this.

Reply:
It was my bad. My home laptop was using tabs and I didn't notice before submitting.